history/cultures/bleeding_hollow.txt

```diff
@@ -35,8 +35,9 @@
 	join_era = culture_era_early_medieval
 }
 605.1.1 = {
-	discover_innovation  = innovation_burhs 
+	discover_innovation = innovation_burhs 
 	#
 	discover_innovation = innovation_baliffs 
 	#
+	discover_innovation = innovation_desert_warfare
```

Comment:
Huh? What is the idea behind it?
Considering their home region they should have jungle warfare.

Reply:
ya my bad